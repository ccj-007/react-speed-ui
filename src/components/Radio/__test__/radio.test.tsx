import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Radio } from '../../index';

describe('test Radio component', () => {
  it('should render', () => {
    const { asFragment } = render(<Radio />);
    expect(asFragment()).toMatchSnapshot();
  });
});
