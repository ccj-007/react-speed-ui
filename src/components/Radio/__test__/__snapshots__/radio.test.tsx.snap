// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Radio component should render 1`] = `
<DocumentFragment>
  <div
    class="speed-radio"
  >
    <input
      class="speed-radio-input"
      id="speed-radio-undefined"
      type="radio"
    />
    <label
      class="speed-radio-label"
      for="speed-radio-undefined"
    />
  </div>
</DocumentFragment>
`;
